plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -1066,7 +1060,7 @@ def run(
             disable=not self.verbose,
             desc="Particles on grid",
             unit="particles",
-            bar_format="{l_bar}{bar}{n:.1e}/{total:.1e} {unit}",
+            bar_format="{l_bar}{bar}{n:.1e}/{total:.1e} {unit}",  # noqa
```

Comment:
```suggestion
            bar_format="{l_bar}{bar}{n:.1e}/{total:.1e} {unit}",  # noqa: FS003
```
If we add the error code, it will still do the quality assurance checks for everything else.

Reply:
Oh nice!